apps/etterlatte-saksbehandling-ui/client/src/shared/components/maanedVelger/ControlledMaanedVelger.tsx

```diff
@@ -46,6 +47,20 @@ export const ControlledMaanedVelger = <T extends FieldValues>({
     },
   } as UseMonthPickerOptions)
 
+  useEffect(() => {
+    if (field.value === undefined) setSelected(undefined)
```

Comment:
Kanskje heller bruke `!!` for å sjekke etter tom verdi? Antar det også skal regnes som ugyldig her.